### PR TITLE
MM-47594 : E2Es for Forwarding post to new DM

### PR DESCRIPTION
**Pull request by AbhinavVihan**
#### Summary
E2Es for Forwarding post to new DM

#### Ticket Link
Fixes: https://github.com/mattermost/mattermost-server/issues/21413
JIRA: https://mattermost.atlassian.net/browse/MM-47594

#### Related Pull Requests
None

#### Screenshots
None

#### Release Note

```release-note
E2Es for Forwarding post to new DM
```

### CODE REVIEW

**Review by michelengelen — APPROVED** [review]
lgtm

**Review by mylonsuren — APPROVED** [review]
LGTM 🚀

File: e2e/cypress/tests/integration/message_forwarding/forward_post_to_new_dm_spec.ts [hunk 1]
```diff
@@ -0,0 +1,210 @@
...
+        cy.get('#channelHeaderTitle', {timeout: TIMEOUTS.HALF_MIN}).should('be.visible').should('contain', dmChannel.display_name);
+
+        // * Assert post has been forwarded
+        verifyForwardedMessage({post: testPost, comment: '', showMore: false});
+
+        // # Go to 1. public channel
+        cy.visit(`/${testTeam.name}/channels/${testChannel.name}`);
+    });
+
+    it('MM-T5215_1 Forwarding post to new DM replying on a post', () => {
```
Comment (jgilliam17) [inline e2e/cypress/tests/integration/message_forwarding/forward_post_to_new_dm_spec.ts:116]:
```suggestion
    it('MM-T5215_2 Forwarding post to new DM replying on a post', () => {
```

**Comment by jgilliam17** [comment]
Thank you @AbhinavVihan for the new test
Both steps are failing - see full report [here](https://mattermost-cypress-report.s3.amazonaws.com/279934-6e5c8c8-onprem-ent-webapp-pr-11827-mm-ee-test:6e5c8c8/mochawesome.html)
 Can you please take a look and fix - thanks 🙂

**Comment by AbhinavVihan** [comment]
@jgilliam17, the failing test was fixed,
Screenshot from 2022-12-26 13-28-59
Thank you

**Comment by jgilliam17** [comment]
Thanks @AbhinavVihan 
Unfortunately test is still failing see report [here](https://automation-dashboard.vercel.app/cycle/280798) 
Screen Shot 2022-12-27 at 12 21 50 PM

**Comment by michelengelen** [comment]
@AbhinavVihan anything we can help with to finish this?

**Comment by AshishDhama** [comment]
@jgilliam17 it's ready for round 2

**Comment by lieut-data** [comment]
Heads up that as part of our efforts to move to a monorepo, we're closing out a number of older pull requests like this one to help streamline the effort.

If you'd like to preserve these changes -- even if you're not the original author! -- feel free to resubmit the pull request against the monorepo once it's ready. You can subscribe to [mattermost-server-issue-22420](https://www.github.com/mattermost/mattermost-server/issues/22420) for status updates on this effort.

**Pull request closed without merging**